### ISSUE TITLE
Adding a New Language and Extending Previous Languages

### ISSUE THREAD

**Issue by mbach138** [unitaryai/detoxify#33]
How would one add a new language to the existing set?  How would one extended what is already there with further examples?

**Comment by laurahanu** [comment]
If you have a sufficient number of labelled examples you should be able to finetune the `multilingual` model directly (might want to check if XLM-Roberta was trained on your language). Details on how to train this model are in the README, you might need to create a data loader for your new dataset. 

If you don't have enough labelled examples, you could translate the Jigsaw datasets used into the new language and retrain the model, although you would probably need to create a labelled test set to check the performance.

Hope this helps!
